service/src/adapters/line.ts

```
@@ -70,7 +70,7 @@ function determineContentType(message: any): string {
       return 'video/mp4'
     case 'file':
       // Try to determine from file extension
-      const extension = getFileExtension(message).toLowerCase()
+      const extension = getFileExtension(message.file).toLowerCase()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)